aws/import_aws_glue_catalog_database_test.go

```diff
@@ -0,0 +1,29 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+)
+
+func TestAccAWSGlueCatalogDatabaseVault_importBasic(t *testing.T) {
```

Comment:
Nitpick: Dang, sorry I didn't notice this copypasta earlier - just needs `Vault` removed